aviatrix/resource_aviatrix_gateway.go

```diff
@@ -412,6 +456,34 @@ func resourceAviatrixGateway() *schema.Resource {
 				Computed:    true,
 				Description: "FQDN gateway lan interface id.",
 			},
+			"public_subnet_filtering": {
```

Comment:
Maybe change to "enable_public_subnet_filtering" or "public_subnet_filtering_enabled"?